composer.json

```diff
@@ -1,5 +1,12 @@
 {
-  "require": {
-    "ext-curl": "*"
-  }
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/CiviMRF/CMRF_Abstract_Core.git"
```

Comment:
The package `civimrf/cmrf_abstract_core` is [registered in Packagist](https://packagist.org/packages/civimrf/cmrf_abstract_core) so this shouldn't be necessary.